Lib/test/test_fstring.py

```diff
@@ -1198,6 +1198,10 @@ def test_invalid_syntax_error_message(self):
         with self.assertRaisesRegex(SyntaxError, "f-string: invalid syntax"):
             compile("f'{a $ b}'", "?", "exec")
 
+    def test_with_two_commas_in_format_specifier(self):
+        error_msg = "Cannot specify ',' with ','."
```

Comment:
Same issue here with a regex.

Reply:
Sure. I will add them, and use `re.escape`